plugins/scaffolder-react/api-report-alpha.md

```diff
@@ -140,6 +140,9 @@ export type ScaffolderPageContextMenuProps = {
   onActionsClicked?: () => void;
   onTasksClicked?: () => void;
   onCreateClicked?: () => void;
+  onTemplateFiltersClicked?: () => void;
+  onTemplateGlobalsClicked?: () => void;
+  additionalItems?: ReactNode[];
```

Comment:
Think we should pick one of these API's here, not both.

Reply:
Can you elaborate on what you meant with this comment?